tamr_unify_client/models/dataset/resource.py

```diff
@@ -59,3 +65,14 @@ def records(self):
         with self.client.get(self.api_path + "/records", stream=True) as response:
             for line in response.iter_lines():
                 yield json.loads(line)
+
+    def status(self) -> DatasetStatus:
+        """Retrieve this datast's streamability status.
+
+        :return: Dataset streamability status.
+        :rtype: DatasetStatus
```

Comment:
```
:class:`~tamr_unify_client.models.dataset_status.DatasetStatus`
``` 
instead of just `DatasetStatus` so that docs render a link to the class.

Reply:
is that 
`:rtype: :class: ~tamr_unify_client.models.dataset_status.DatasetStatus`?

Reply:
oic, it's
```
:rtype: :class:`tamr_unify_client.models.dataset_status.DatasetStatus`
```

Reply:
or do I need the `~` for something?

Reply:
the `~` shortens the rendered link to just the last thing so `DatasetStatus` instead of `tamr_unify_client.models.dataset_status.DatasetStatus`. We've been using `~`/shortened links unless there's a good reason not to.

Reply:
so that would be:
```
:rtype: :class:~`tamr_unify_client.models.dataset_status.DatasetStatus`
```
?

Reply:
```:rtype: :class:`~tamr_unify_client.models.dataset_status.DatasetStatus````. So `~` inside the ```````.

Reply:
You can install the docs requirements (`pip install -r doc/requirements`) and then run `make html` from within the `doc` dir and open up the resulting `_build/html/index.html` in your web browser if you want to see it render.

Reply:
👍